strax/processing/hitlets.py

```diff
@@ -244,6 +244,7 @@ def get_single_hitlet_data(hitlet, records, prev_r, next_r):
             r = records[next_r_i]
             data, (start, p_end_i) = _get_thing_data(hitlet, r)
             if not start:
+                print(data, start, p_end_i)
```

Comment:
Intended?

Reply:
Yes, but I agree I should put some additional text.